core/templates/dev/head/pages/exploration_player/conversation_skin_directive.html

```diff
@@ -210,10 +210,18 @@
                                 is-community-owned="exp.community_owned"
                                 ng-attr-open-in-new-window="<[isIframed ? 'true' : undefined]>">
       </exploration-summary-tile>
-
+      <collection-summary-tile ng-if="collectionId"
```

Comment:
Actually, I withdraw this. I seem to remember that Ed added some logic such that only one exploration recommendation shows up when you're in the context of a collection.